scripts/makedatacount/process_mdc_logs.sh

```diff
@@ -0,0 +1,167 @@
+#! /bin/bash
```

Comment:
It's designed to process archived files. We (Harvard) will be processing only files that have been archived so we won't be running the daily script. Since I don't see any mention of the script that archives the logs I didn't know if this should be documented for anyone but us.  The daily script doesn't give us the monitoring and processing control that this new script can give.
@landreev - do you have an opinion of how the two scripts should be documented, if at all? The other script isn't checked into the repo, which I think it should be.

Reply:
@stevenwinship 
Going through my GitHub mentions, just saw the above now, sorry. 

I don't have a super strong opinion. But generally, things that are super Harvard/IQSS-specific should not be checked in the main repo. For our own housekeeping purposes we can check things in under the https://github.com/IQSS/dataverse.harvard.edu repo. Or simply document it in the "Custom Scripts and Basic Tasks on the Production Servers" Google doc. 

As @qqmyers mentioned, in some cases it can be useful to check in scripts like this and document them, as potential examples that other instances may find useful. I'm not sure about this specific script though - it may just be too exotic for that purpose. For example, having to do this on two different server nodes is a complication that may be unique to us.

Reply:
Will be moving the script to the Harvard dataverse repo

Reply:
@stevenwinship - does that mean this PR should be closed? (It's 'Ready for Review' right now.)

Reply:
No. The script was moved but this fix updates the counter-processor and the nightly script and docs. This is needed to include the counter-processor bug fix that prompted the new version that dataverse will use.